app/helpers/markdown_helper.rb

```diff
@@ -3,15 +3,30 @@ def render_markdown_to_html(markdown_content)
     return "" if markdown_content.blank?
 
     html = Kramdown::Document.new(markdown_content).to_html
-    return html.html_safe unless html.include?("href")
 
     doc = Nokogiri::HTML.fragment(html)
+    doc = update_class(doc, "h2", "govuk-heading-s")
+    doc = update_class(doc, "p", "govuk-body")
+
+    return doc.to_html.html_safe unless html.include?("href")
```

Comment:
the check for "href" was earlier to avoid doing any Nokogiri parsing if it's not required (which did mean that there were no links that need outbound link mangling).

I think the best thing here would be to do something like `return html.html_safe unless html.include?(/href|<p\b|<h2\b/)` at line 6 - or drop the check entirely if it's unnecessary optimisation.

Reply:
This has been been updated and i have added a few more test around this around of the code